core/src/worker/workflow/mod.rs

```diff
@@ -1005,6 +1005,18 @@ fn validate_completion(
                 });
             }
 
+            // Any commands after a terminal command should be ignored
+            if let Some(term_cmd_pos) = commands.iter().position(|c| c.is_terminal()) {
+                let remaining = commands.split_off(term_cmd_pos + 1);
+                if !remaining.is_empty() {
+                    warn!(
+                        "Activation included commands after a terminal workflow command. These \
```

Comment:
Can you confirm query results are still ok?

Reply:
Not sure we want the warning here, maybe debug.
We don't warn in lang today, we accept that this can happen if workflow is concurrent.

Reply:
Yeah, good point. I have to double check if that scenario is even possible, but either way I'll protect for it.

Reply:
I think the warning is worth keeping. Pretty much by definition users are writing somewhat sloppy WF code if they are doing this, but I don't hugely care. We can git rid of it, I'm just not sure exactly who getting rid of it benefits.

Reply:
I got rid of it, not like anyone will use it anyway lol